packages/flutter/lib/src/material/text_field.dart

```diff
@@ -115,6 +115,7 @@ class TextField extends StatefulWidget {
     this.onSubmitted,
     this.inputFormatters,
     this.enabled,
+    this.scrollInsets = const EdgeInsets.all(20.0),
```

Comment:
`scrollPadding` was probably the better name for this.

(The Padding widget also takes an EdgeInsets object as a parameter, so that would be nothing unusual.)